ops/ansible/roles/cloudwatch-agent-instrumented/templates/cwagent.json.j2

```diff
@@ -12,27 +12,64 @@
           "usage_idle",
           "usage_iowait",
           "usage_user",
-          "usage_system"
+          "usage_system",
+          "usage_active" 
+
```

Comment:
Seems like an extraneous blank line here.
```suggestion
```

Reply:
I'll strike the line